src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs

```diff
@@ -190,7 +192,12 @@ public void AkkaProtocolTransport_must_guarantee_at_most_once_delivery_and_messa
                         new FailureInjectorTransportAdapter.Drop(0.1, 0.1)));
             AwaitCondition(() => mc.IsCompleted && mc.Result, TimeSpan.FromSeconds(3));
 
-            var here = Here;
+            IActorRef here = null;
+            AwaitCondition(() =>
```

Comment:
`Here` property uses `ActorSelection` that can return a null `Subject` that causes NRE down the line (caused by `EmptyLocalActorRef`). Using `AwaitCondition` here to force the spec to wait until `Here` returns a valid `IActorRef`

Reply:
LGTM - how many times did you run this one successively without failures?

Reply:
1000 times minimum

Reply:
That works for me